src/common/WorkQueue.cc

```diff
@@ -124,6 +124,7 @@ void ThreadPool::worker(WorkThread *wt)
 	void *item = wq->_void_dequeue();
 	if (item) {
 	  processing++;
+          _wq_processing[wq]++;
```

Comment:
@jinweiyi (NIT) Please consider to keep PR title < 50 characters for future PRs.
https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#title-of-pull-requests-and-title-of-commits